tests/acceptance/standard/test_group_avatar.py

```diff
@@ -0,0 +1,292 @@
+        try:
+            os.chdir(temp_dir2)
+
+            subdir = "test_images"
+            os.makedirs(subdir)
+            relative_subdir_path = os.path.join(subdir, "subdir_avatar.png")
+            shutil.copy2(self.test_image_path, relative_subdir_path)
+
+            config_relative_subdir = f"""
+            projects_and_groups:
+              {group.full_path}/*:
+                group_settings:
+                  avatar: "{relative_subdir_path}"
+            """
+            run_gitlabform(config_relative_subdir, group)
+
+            group = group.manager.get(group.id)
+            assert group.avatar_url is not None
+
+        finally:
+            os.chdir(original_cwd)
+            shutil.rmtree(temp_dir2)
+
+    def test__group_avatar_delete(self, group):
+        # First ensure there's an avatar to delete by setting one
+        config_set = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config_set, group)
+
+        # Refresh group data to get current state
+        group = group.manager.get(group.id)
+
+        # Verify that avatar exists
+        assert group.avatar_url is not None, "Avatar should exist after setting it"
+
+        # Delete the avatar
+        config_delete = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: ""
+        """
+        run_gitlabform(config_delete, group)
+
+        # Refresh group data
+        group = group.manager.get(group.id)
+
+        # Verify avatar is removed
+        assert group.avatar_url is None or "gravatar" in group.avatar_url
+
+    def test__group_avatar_file_not_found_should_fail(self, group):
+        # Test handling of non-existent avatar file path - should fail
+        nonexistent_path = "/path/to/nonexistent/image.png"
+
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{nonexistent_path}"
+        """
+
+        # GitLabForm catches exceptions
+        with pytest.raises(SystemExit) as exc_info:
+            run_gitlabform(config, group)
+
+        # Verify it's the expected exit code for processing errors
+        assert exc_info.value.code == 2
+
+    def test__group_avatar_relative_file_not_found_should_fail(self, group):
+        # Test handling of non-existent relative avatar file path - should fail
+        original_cwd = os.getcwd()
+        temp_dir = tempfile.mkdtemp()
+
+        try:
+            os.chdir(temp_dir)
+
+            # Use non-existent relative path
+            nonexistent_relative_path = "images/nonexistent_avatar.png"
+
+            config = f"""
+            projects_and_groups:
+              {group.full_path}/*:
+                group_settings:
+                  avatar: "{nonexistent_relative_path}"
+            """
```

Comment:
In this test case, do we need those logic around `os.getcwd`, `tempfile.mkdtemp`? Not sure how it's being used in the tests. Could you please clarify what I'm missing?